### PR TITLE
✨ Whats-new pages fixes

### PR DESCRIPTION
**Pull request by joshbermanssw**
For PBI #2189 
- Removes broken links on /whats-new/tinacloud page 
- Adds glass effect to cards on both /whats-new pages 
- Changes wording from 'See More' to 'See more on GitHub' to indicate external link 
- Adds link to all newsletter pages from both /whats-new pages 

Screenshot 2024-09-16 at 1 14 01 PM
**Figure: 🍸 Glass effect on cards**

### CODE REVIEW

File: components/ui/DynamicLink.tsx [hunk 1]
```diff
@@ -10,12 +10,9 @@ interface DynamicLinkProps extends ExtraProps {
...
-    // Because {children} contains <a> tags we need to wrap it in a div to pass className attributes in. If we inserted into the <Link> it doesnt get passed in. 
-    <div className="cursor-pointer">
-      {/* Work around for using legacyBehavior so that anchors in HTML in MDX files do not cause legacy errors */}
-      <Link href={href} {...props} legacyBehavior>
-        {children}
-      </Link>
-    </div>
+    <Link href={href} {...props} legacyBehavior>
+      <a className="cursor-pointer">{children}</a>
+    </Link>
```
Comment (bradystroud) [inline components/ui/DynamicLink.tsx:15]: Can we fix up `legacyBehavior`

I know the code was already like that, but i think we should fix things like this as we go :)

**Review by bradystroud — CHANGES_REQUESTED** [review]

File: components/ui/DynamicLink.tsx (hunk 1 above)
Comment (joshbermanssw) [reply components/ui/DynamicLink.tsx]: apoc, done

**Review by isaaclombardssw — APPROVED** [review]
Code LGTM.

Could improve the UX with some styling change (darkening) of these on hover - see current UX below.

File: pages/whats-new/tinacms.tsx [hunk 2]
```diff
@@ -48,14 +49,22 @@ const Tinacms = ({ items }) => {
...
+            href="https://github.com/tinacms/tinacms/blob/main/packages/tinacms/CHANGELOG.md"
+            className="flex items-center justify-center"
+          >
+            See more on GitHub <FaGithub className="ml-2" />
+          </Link>
+          <Link
+            href="https://us20.campaign-archive.com/home/?u=1fea337bee20e7270d025ea8a&id=c1062536a1"
+            className="flex items-center justify-center"
+          >
+            See Newsletters <FaNewspaper className='ml-2' />
```
Comment (bradystroud) [inline pages/whats-new/tinacms.tsx:62]: Not a big one, but in the future I would prefer avoiding hardcoded URLs like this - They should be editable in TinaCMS

**Review by bradystroud — APPROVED** [review]

File: pages/whats-new/tinacms.tsx (hunk 2 above)
Comment (joshbermanssw) [reply pages/whats-new/tinacms.tsx]: 🫡

**Pull request merged**